src/core/api/index.js

```diff
@@ -1,4 +1,5 @@
 /* global fetch */
+/* eslint-disable arrow-body-style */
```

Comment:
oh, I see what you mean. Well, I like doing things like `someArray.filter((value) => value === true)`. I wouldn't want to give that up.